tests/solidity_test_helpers/ControllerUser.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/IControlled.sol';
+
+
+contract ControllerUser is IControlled {
+    address public suicideFundsDestination;
+    address public updatedController;
+
+    function setController(IController _controller) public returns(bool) {
```

Comment:
Shouldn't these be explicitly returning true since the return type is set to bool?  (Also, small nit: there should be a space after `returns`.)

Reply:
Fixed.